cmake/checks/check_01_cxx_features.cmake

```diff
@@ -278,6 +278,7 @@ IF(NOT DEFINED DEAL_II_WITH_CXX14 OR DEAL_II_WITH_CXX14)
     # https://llvm.org/bugs/show_bug.cgi?id=16876
     #
     PUSH_CMAKE_REQUIRED("${DEAL_II_CXX_FLAGS_DEBUG}")
+    PUSH_CMAKE_REQUIRED("-Wno-error")
```

Comment:
We only run this part of the script with GCC and Clang, right? Or do we need to worry about compilers that use different flags for this?

Reply:
No, this is run by all the compilers.

Reply:
ICC-19 does seem to have a problem with this.

Reply:
Ah, darn :-( @tamiko -- your take on this?

Reply:
MSVC 2019 doesn't seem to like it, but is fine with '-g'. Let me get back to this.